source/agora/test/Flash.d

```diff
@@ -220,7 +220,11 @@ public class TestFlashNode : ThinFlashNode, TestFlashAPI
         auto peer = new RemoteAPI!TestFlashAPI(tid, timeout);
         this.known_peers[peer_pk] = peer;
         if (this.known_channels.length > 0)
+        {
             peer.gossipChannelsOpen(this.known_channels.values);
+            peer.gossipChannelUpdates(this.channel_updates.byValue
+                .map!(updates => updates.byValue.array).joiner.array);
```

Comment:
That's a lot of array call... Couldn't the function take a range ?
Additionally, I assume those will not be combined into a single request ?